exec/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 201i7 The Kubernetes Authors.
```

Comment:
we use this file in k/k: https://github.com/kubernetes/kubernetes/blob/master/hack/verify-boilerplate.sh